slyr_community/converters/symbols.py

```diff
@@ -3183,7 +3183,12 @@ def append_CharacterMarkerSymbolLayerAsSimpleMarker(symbol,
                 elif marker_type == QgsSimpleMarkerSymbolLayerBase.AsteriskFill:
                     simple_size = (1.3632460 * layer.size) / 2.0
 
-            simple_size *= conversion_properties.get('shape_size_factor', 1)
+            try:
+                simple_size *= conversion_properties.get('shape_size_factor', 1)
+            except NameError as e:
+                if "simple_size" in str(e):
+                    raise NotImplementedException(f"Unknown size multipyer for {marker_type}.") from e
```

Comment:
Do you mean this by "raw assert"?

```suggestion
                    assert False
                    
```